clients/mobile/src/authentication/services/authErrorHandler.ts

```diff
@@ -0,0 +1,42 @@
+export const handleAuthError = (error: unknown): string => {
+  let errorMessage = 'Unknown error occurred.'
```

Comment:
you can return from the case instead of this unnecessary let

Reply:
Removed Let and used return.[Commit](https://github.com/CraftsmenLtd/BloodConnect/pull/197/commits/cc280e8a37f9c1825144ff7ca528169d72298a0a)